src/js/views/patients/worklist/worklist-list.scss

```diff
@@ -24,6 +24,12 @@
   width: 16px;
 }
 
+.worklist-list__details {
+  color: $black-60;
+  padding: 16px 0;
+  width: 16px;
```

Comment:
Same here with injecting a fixed width inside the percentage columns.    It's possible that this should really go inside the meta since these spacing wise are really grouped together.  Makes the headings a little more interesting, but probably just a `padding-left: 24px` or something like that

Reply:
I'm thinking it should either live in the beginning of the meta cell or absolutely positioned on the right side of the action name cell. It would avoid wonkiness with pixels/percents in the table.

Reply:
I can't remember what breaks the white-space ellipsis stuff on the action. Would an absolute positioned element mess with that?

Reply:
As long as the container's padding accounts for the absolutely positioned element, putting it in the padded area, the ellipsis should work as expected.